addons/website_slides/static/src/js/activity.js

```diff
@@ -1,45 +1,47 @@
 odoo.define('website_slides.Activity', function (require) {
-"use strict";
+    "use strict";
 
-var field_registry = require('web.field_registry');
+    const components = {
+        Activity: require('mail/static/src/components/activity/activity.js'),
+    };
 
-require('mail.Activity');
+    const { patch } = require('web.utils');
 
-var KanbanActivity = field_registry.get('kanban_activity');
+    patch(components.Activity, 'website_slides/static/src/components/activity/activity.js', {
 
-function applyInclude(Activity) {
-    Activity.include({
-        events: _.extend({}, Activity.prototype.events, {
-            'click .o_activity_action_grant_access': '_onGrantAccess',
-            'click .o_activity_action_refuse_access': '_onRefuseAccess',
-        }),
+        async willStart() {
+            await this._super(...arguments);
+            if (this.activity && this.activity.creator && !this.activity.creator.partner) {
+                await this.activity.creator.fetchPartner();
+            }
+        },
```

Comment:
Don't block `willStart` in new code. Component should be able to handle incomplete information. And refresh itself when it gets it. For example this can simply be in template to disable the buttons while partner of creator isn't known.

We don't want to block the render of the whole chatter while this RPC is pending.